api/tests/Behat/features-v2/user-management/create-user.feature

```diff
@@ -0,0 +1,37 @@
+@v2 @admin-user-create @admin-user
+Feature: Admin - User Create
+
+@super-admin
+  Scenario: A super admin user creates a deputy user
+    Given a super admin user accesses the admin app
+    When I add a new lay deputy user
+    Then I see that an activation email has been sent to the user
+    When I search for the newly created user
+    Then I can see the user as non active in search results
+    When I resend activation email
+    Then I see that activation link has been sent
+
+@super-admin
+Scenario: A super admin creates admin users
+    Given a super admin user accesses the admin app
+    When I add each of the appropriate user types for a super admin
```

Comment:
```suggestion
    When I add each of the available user types for a super admin
```

Feels like this is a bit more explicit about the fact adding users with specific roles are limited to different admin type? What do you think?

Reply:
Sure. I'm down with that!